requirements.txt

```diff
@@ -8,4 +8,4 @@ gpsoauth==0.3.0
 coveralls==1.1
 werkzeug==0.11.10
 sqlalchemy==1.0.14
--e git+https://github.com/tejado/pgoapi.git@0811db23d639039f968a82e06c7aa15a0a5016b6#egg=pgoapi
+-e git+https://github.com/keyphact/pgoapi.git#egg=pgoapi
```

Comment:
@Brewfasa first this was not the right place to comment. Second: try these things: http://www.pandastack.com/unable-to-find-vcvarsall-bat-how-to-fix-on-windows/